core/backend/src/internal/ChannelAdmin.ts

```diff
@@ -73,7 +73,7 @@ class ChannelAdmin implements ChannelControl {
 
     const deniedChannel = this._deniedModels.get(modelId);
     if (undefined !== deniedChannel)
-      throw new Error(`channel "${deniedChannel}" is not allowed`);
+      throw new IModelError(ChannelStatus.NotAllowed, `channel "${deniedChannel}" is not allowed`);
```

Comment:
wonder if we should use ITwinError instead

Reply:
definitely. We should stop using IModelError entirely. Error numbers are not useful.